### DISCUSSION TITLE
FYI A Pic of my trailer rear camera implementation

### DISCUSSION THREAD

**Discussion by jmfloyd** [easytarget/esp32-cam-webserver#162] [Show and tell]
![RearCamera-2](https://user-images.githubusercontent.com/1490471/120181550-8f2a0500-c250-11eb-86a6-c80244a3da18.jpg)

**Comment by easytarget** [comment]
Nice! I assume you have brought the serial port out on the pin connector for easy programming etc.

**Comment by moelect** [comment]
Just getting started with a esp32-cam and trying to figure a mounting means with a board that has no screw holes.  Very nice installation by the way!

**Comment by jmfloyd** [comment]
My approach for my camping  trailer,  was to use foam and adjust the thickness so that camera lens sits snug against the clear top.  Use hard spacers to lock the main board from moving sideways.Thanks for the comment.John

**Reply by moelect** [reply]
Thanks for the great info!  I'm going to do just what you recommended, good plan, looking for spacers now.
